nylas/client/client.py

```diff
@@ -256,7 +258,8 @@ def _get_resources(self, cls, extra=None, **filters):
                 postfix
             )
 
-        url = str(URLObject(url).add_query_params(filters.items()))
+        ts_filters = timestamped_data(filters, cls.datetime_attrs)
```

Comment:
ts_filters is a slightly confusing name here. It makes it seem like it's only timestamp filters. Maybe converted_filters or request_filters?